src/Elastic.Apm.AspNetCore/Helpers/AgentConfigHelper.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+using System.Text;
+using Elastic.Apm.Config;
+using Elastic.Apm.Logging;
+using Elastic.Apm.Model;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+
+namespace Elastic.Apm.AspNetCore.Helpers
+{
+	/// <summary>
+	/// Simplifies the way to determine if we should collect request field's body during trx, errors etc.
+	/// </summary>
+    public static class AgentConfigHelper
+    {
+		/// <summary>
+		/// Returns weather we should collect the request body in case of error (accoridng to the Apm configuration)
```

Comment:
nit: small typo: `accoridng` -> `according `

Reply:
I just found out that for some reason my VS2019 installation didn't have a spell checker installed. Going over the files to verify there are no more typos.

Reply:
👍  No worries, this happens to me even with spell checker 😄

Reply:
Fixed.